api/inc/debug_exports.h

```diff
@@ -19,11 +19,17 @@
 
 #include <stdint.h>
 
+
+#define UVISOR_DEBUG_BOX_MAGIC      (0xDB4807DB)
```

Comment:
Please place this magic with all the other magic. Also, generate the magic in the same way as the other magics.